src/Microsoft.Management.Deployment/PackageVersionInfo.cpp

```diff
@@ -138,4 +139,24 @@ namespace winrt::Microsoft::Management::Deployment::implementation
     { 
         return m_packageVersion; 
     }
+    winrt::Microsoft::Management::Deployment::PackageInstallerInfo PackageVersionInfo::GetApplicableInstaller(InstallOptions options)

```

Comment:
Nit: let's move this method above GetRepositoryPackageVersion() to let interface methods stay together. And internal methods stay below.